src/components/login/register.jsx

```diff
@@ -0,0 +1,102 @@
+import React from "react";
+import "./login.scss";
+import { toast } from "react-toastify";
+import { useDispatch } from "react-redux";
+import useAxios from "../../hooks/useAxios";
+import { authActions } from "../../redux/authSlice";
+
+function Register() {
```

Comment:
[Optional Suggestion] I  appreciate the addition of the register button. However, after registering, It would be better if it navigates to the Login page or main page.
![Screenshot from 2023-09-04 14-10-54](https://github.com/sumon766/frontend-doctor-appointment/assets/86189460/bd8a708f-2fd7-4e2f-8403-c941908e6677)